script/releases/v0.1-eigenpod-example/2-multisig.s.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "zeus-templates/templates/OpsTimelockBuilder.sol";
+
+import {ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import {IUpgradeableBeacon} from "script/interfaces/IUpgradeableBeacon.sol";
+import "src/contracts/pods/EigenPodManager.sol";
+
+contract QueueEigenPodAndManager is OpsTimelockBuilder {
+
+    using MultisigCallUtils for MultisigCall[];
+    using SafeTxUtils for SafeTx;
+
+    MultisigCall[] internal _executorCalls;
+
+    function queue(Addresses memory addrs, Environment memory env, Params memory params) public override returns (MultisigCall[] memory) {
```

Comment:
Weird that this method in particular is public rather than the internal pattern the other two scripts use

Reply:
ah, i see now why it's public, used in script 3.

eesh, that feels like a weird pattern!

Reply:
2 main reasons:
1. this is the only time we do this pattern (deploy a script and call it from another script)
2. deploys/calls like this should be minimized because then you have to be careful how/when you use start/stop broadcast

Reply:
would much rather make this an inheritance relationship somehow